assets/js/translator.js

```diff
@@ -65,13 +65,18 @@ if(modeEnabled('translation')) {
         }
 
         function setupWebpageTranslation() {
+            if(window.location.href.indexOf('&qP=') != -1) {
```

Comment:
In fact, right now translation gets triggered automatically on page load if there's text present, ideally we could easily modify that code to make it also do this.